cmd/eskip/load.go

```diff
@@ -126,7 +127,14 @@ func printCmd(a cmdArgs) error {
 			}
 		}
 
+		if sortPredicate {
+			for _, route := range lr.routes {
+				sort.Slice(route.Predicates, func(i, j int) bool { return i < j })
```

Comment:
Should there be some unit tests to check whether such sort actually works?

Reply:
yes, that's in progress.

Reply:
Implemented in `eskip/string_test.go`